src/Components/Containers/DataGrid/UpDefaultCellFormatter.tsx

```diff
@@ -122,6 +122,11 @@ export class UpCellFormatter extends React.Component<UpCellFormatterProps, {}>{
 
     render() {
         if (this.props.column.formatter != null) {
+            if(this.props.column.getFormatterProps != null)
```

Comment:
let additionalProps = this.props.column.getFormatterProps && this.props.column.getFormatterProps(this.props.value[this.props.column.field]);
return this.props.column.formatter.format(this.props.value, this.props.column, additionalProps || {});